src/agora/dofft.cpp

```diff
@@ -378,16 +408,18 @@ Event_data DoIFFT::launch(size_t tag)
     // IFFT scaled results by OFDM_CA_NUM, we scale down IFFT results
     // during data type coversion
     simd_convert_float_to_short(ifft_out_ptr, socket_ptr, cfg->OFDM_CA_NUM,
-        cfg->CP_LEN, cfg->OFDM_CA_NUM);
+        cfg->CP_LEN, cfg->OFDM_CA_NUM / std::sqrt(cfg->BF_ANT_NUM * 1.f));
```

Comment:
We should avoid doing division here due to performance issue. Maybe we can compute `cfg->OFDM_CA_NUM / std::sqrt(cfg->BF_ANT_NUM * 1.f)` in the constructor of doIFFT?

Reply:
Done!